rspace/src/main/scala/coop/rchain/rspace/HotStoreAction.scala

```diff
@@ -0,0 +1,21 @@
+package coop.rchain.rspace
+
+import coop.rchain.rspace.internal.{Datum, WaitingContinuation}
+
+sealed trait HotStoreAction
+final case class InsertData[C, A](key: Blake2b256Hash, channel: C, data: List[Datum[A]])
```

Comment:
Let's use the more general `Seq` instead of `List` in case we'd like to move to the better performing `Vector`